src/main.c

```diff
@@ -1987,9 +2046,12 @@ static void setup_configurators(void)
 int main(int argc, char **argv)
 {
     const char *cmd = argv[0], *opt_config_file = H2O_TO_STR(H2O_CONFIG_PATH);
-    int error_log_fd = -1;
+    int n, error_log_fd = -1;
+    size_t num_threads = h2o_numproc();
```

Comment:
Nitpick, but we might want to name the variable `num_procs`, because it indicates the number of CPU cores, not the number of threads in any particular way.